ui/nuxeo-format-behavior.js

```diff
@@ -137,17 +137,22 @@ export const FormatBehavior = [
       if (!type) {
         return;
       }
-      const key = `label.document.type.${type.toLowerCase()}`;
-      const value = this.i18n(key);
-      return value === key ? type : value;
+      return this.i18nWithPrefix('label.document.type', type.toLowerCase());
+    },
+
+    /**
+     * Returns the label for the given document permission
+     */
+    formatPermission(permission) {
```

Comment:
I can understand the usage and purpose of `i18nWithPrefix` but, for criteria coherency, I'm not sure if we need to refactor this code.
Probably our new method `formatPermission` could be something like:

```suggestion
    formatPermission(permission) {
      if (!permission) return;
      return this.i18n(`label.security.permission.${permission.substring(0, 1).toLowerCase() + permission.substring(1)}`);
    }
```